test/integration/dialects/sqlite/dao.test.js

```diff
@@ -61,26 +64,242 @@ if (dialect === 'sqlite') {
           projects: [
             { dateField: new Date(1990, 5, 5) }
           ]
-        }, { include: [this.Project]}).bind(this).then(function () {
+        }, { include: [this.Project]}).then(() => {
           return this.User.findAll({
             include: [this.Project]
           }).get(0);
-        }).then(function (user) {
+        }).then(user => {
           expect(user.projects[0].get('dateField')).to.be.an.instanceof(Date);
           expect(user.projects[0].get('dateField')).to.equalTime(new Date(1990, 5, 5));
         });
       });
     });
 
+    describe('json', function() {
```

Comment:
I'm not sure... I just copied this test suite from [postgres/dao.test.js#L97](https://github.com/sequelize/sequelize/blob/master/test/integration/dialects/postgres/dao.test.js#L97) and there would be some different test like:
-  `->>` operator in [postgres/dao.test.js#L221](https://github.com/sequelize/sequelize/blob/master/test/integration/dialects/postgres/dao.test.js#L221) 
- `json_extract(...)` method in [sqlite/dao.test.js#L232](https://github.com/sequelize/sequelize/pull/7094/files/966d159346a926739ab9f0a336c8c2b8def9775e#diff-30bcb6f77557dd6d0bb295e7815e0d1fR232)

But there are certainly common tests. So where should we move them?